src/NUnitTestAdapter/NUnit3TestExecutor.cs

```diff
@@ -218,7 +218,6 @@ private void RunAssembly(string assemblyName, IFrameworkHandle frameworkHandle)
                 TestLog.SendErrorMessage("Exception thrown executing tests in " + assemblyName, ex);
             }
             _testRunner.Dispose();
-            Unload();
```

Comment:
Rightly removed. But since it's still called elsewhere, I want to say that the name is confusing. It seems as if we are unloading the NUnitTestAdapter, which is of course impossible. UnloadEngine maybe?

We may also want to unload the engine in a dispose implementation for the adapter.